models/project_repository.go

```diff
@@ -0,0 +1,188 @@
+package models
+
+import (
+	"log"
+
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/project"
+	"github.com/jinzhu/gorm"
+	satoriuuid "github.com/satori/go.uuid"
```

Comment:
I copied this from some code by @kwk. I thought goa uses it's own goa uuid package, no?